app/models/user.rb

```diff
@@ -0,0 +1,33 @@
+require 'bcrypt'
+
+class User
+  include DataMapper::Resource
+
+  has n, :peeps
+
+  property :id, Serial
+  property :email, String, required: true, unique: true
+  property :firstname, String, required: true
```

Comment:
What is the convention for table column names with multiple words in them (eg first name)?